docs/content/direct/packaging.md

```diff
@@ -187,9 +190,11 @@ The image repository is
 
 | image tag | git commit | SHA256 digest |
 | --------- | ---------- | ------------- |
-| 0.20.0    | a2bcaf75dc895dbef5f6cddcf203b0203423a08a | fac5b208a1f691eb5548a7699494bed2ee6b7502e739a2db840f39bbc05b2fd4 |
-| 0.21.0    | dcb232dbcfeb2e90d02315c38e26ba1a380d2cc6 | 3f04a74698d0e59bfe941d6ab016ee438f6a6ea684c0f671c03db0f4d7ee5eee |
-| 0.21.1    | 13fcd6a1c10eb377b20e7b1a6c5162209f70a281 | 0e0873088a2a78e578cf97e486ba5348b8728cdf0a270c6741710a241f12af8e |
+| 0.20.0     | a2bcaf75dc895dbef5f6cddcf203b0203423a08a | fac5b208a1f691eb5548a7699494bed2ee6b7502e739a2db840f39bbc05b2fd4 |
+| 0.21.0     | dcb232dbcfeb2e90d02315c38e26ba1a380d2cc6 | 3f04a74698d0e59bfe941d6ab016ee438f6a6ea684c0f671c03db0f4d7ee5eee |
+| 0.21.1     | 13fcd6a1c10eb377b20e7b1a6c5162209f70a281 | 0e0873088a2a78e578cf97e486ba5348b8728cdf0a270c6741710a241f12af8e |
+| 0.21.2     | 7d297a0d1ea500511612c920f7e241c17b866b72 | 76c475c64218cd69838eadf6a4e453d029ad01ff2bd6f9dc966a4f59ba15bd48 |
+| 0.22.0-rc1 | 3006e78fa523300d79e42bdc2913d905e3f9631c | 0b21f8b7cbd59cfa32199366f25d1c817b1d7cfe9d178678e92ba6500abfa841 |
```

Comment:
@MikeSpreitzer  
Not related to this PR, but I never understood the need to maintain this info in this packaging file. 
1) Who actually uses it?
2) Why can't someone who needs it simply search the tag in git 
3) If for some strange reason we do need to doc the SHA and commit - why not automatically push it to the release notes during release build
In short - I strongly recommend to remove this section (cc: @pdettori )